java-checks/src/main/resources/org/sonar/l10n/java/rules/squid/S3776_java.json

```diff
@@ -0,0 +1,15 @@
+{
+  "title": "Cognitive Complexity of methods should not be too high",
```

Comment:
I generated this before putting the rule in the default profile in RSpec. Will there be a final re-generation before release @benzonico?